Source/Initialization/WarpXInitData.cpp

```diff
@@ -764,7 +764,10 @@ WarpX::InitLevelData (int lev, Real /*time*/)
 
     for (int i = 0; i < 3; ++i) {
 
-        if (B_ext_grid_s == "constant" || B_ext_grid_s == "default") {
+        // Externally imposed fields are only initialized until the user-defined maxlevel_extEMfield_init.
+        // The default maxlevel_extEMfield_init value is the total number of levels in the simulation
+        if ( ( B_ext_grid_s == "constant") && (lev <= maxlevel_extEMfield_init) )
```

Comment:
I do not understand the `<=` here:

As far as I am aware, `lev` goes `0...maxLevel()-1`, no?

Reply:
aho, got it